test/unit/auditor_test.js

```diff
@@ -0,0 +1,20 @@
+"use strict";
+
+let chai = require("chai");
+let sinon = require("sinon");
+let sinonChai = require("sinon-chai");
+let expect = chai.expect;
+chai.use(sinonChai);
+
+import auditor from "../../src/auditor";
+
+describe("default", () => {
+  it("runs axe-core tests", () => {
+    let axeSpy = sinon.spy();
```

Comment:
'let' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).